### PR TITLE
Bug/schema compare apply confirmation

### LINKED ISSUE #5691: Confirm at apply before updating database
Schema compare Apply button should ask for confirmation since the changes caused will be persistent to database

### PR DESCRIPTION
**Pull request by udeeshagautam**
This PR adds a confirmation dialog same as SSDT when apply button is clicked before applying the changes to server. This fixes #5691

Looks like following :
image

Bundled up update of sqltools service version for previous schema compare changes

### CODE REVIEW

File: extensions/schema-compare/src/schemaCompareResult.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ import { Telemetry } from './telemetry';
 import { getTelemetryErrorType } from './utils';
 const localize = nls.loadMessageBundle();
 const diffEditorTitle = localize('schemaCompare.ObjectDefinitionsTitle', 'Object Definitions');
+const appliyConfirmation = localize('schemaCompare.ApplyConfirmation', 'Are you sure that you want to update the target?');
```
Comment (kisantia) [inline extensions/schema-compare/src/schemaCompareResult.ts]: nit: typo - appliyConfirmation

Comment (alanrenmsft) [reply extensions/schema-compare/src/schemaCompareResult.ts]: Are you sure you want to apply changes to the target?

what is the message in SSDT?

File: extensions/schema-compare/src/schemaCompareResult.ts [hunk 2]
```diff
@@ -482,24 +483,31 @@ export class SchemaCompareResult {
...
-				'endTime': Date.now().toString(),
-				'operationId': this.comparisonResult.operationId
+
+			vscode.window.showWarningMessage(appliyConfirmation, { modal: true }, yesString).then(async (result) => {
+				if (result === yesString) {
+					Telemetry.sendTelemetryEvent('SchemaCompareApplyStarted', {
+						'startTime': Date.now().toString(),
+						'operationId': this.comparisonResult.operationId
+					});
+					let service = await SchemaCompareResult.getService('MSSQL');
```
Comment (alanrenmsft) [inline extensions/schema-compare/src/schemaCompareResult.ts]: const wherever possible

File: extensions/schema-compare/src/schemaCompareResult.ts [hunk 3]
```diff
@@ -482,24 +483,31 @@ export class SchemaCompareResult {
...
+					let result = await service.schemaComparePublishChanges(this.comparisonResult.operationId, this.targetEndpointInfo.serverName, this.targetEndpointInfo.databaseName, azdata.TaskExecutionMode.execute);
```
Comment (alanrenmsft) [inline extensions/schema-compare/src/schemaCompareResult.ts]: const

File: extensions/schema-compare/src/schemaCompareResult.ts (hunk 1 above)
Comment (udeeshagautam) [reply extensions/schema-compare/src/schemaCompareResult.ts]: Yeah _that_ threw me off too but its the same msg in SSDT. Should I still remove _that_ ?
image

Comment (alanrenmsft) [reply extensions/schema-compare/src/schemaCompareResult.ts]: @kisantia , native English speaker, please share your thoughts.

Comment (kisantia) [reply extensions/schema-compare/src/schemaCompareResult.ts]: Yes remove "that"

**Pull request merged**